zrml/court/src/lib.rs

```diff
@@ -1293,6 +1304,9 @@ mod pallet {
             }
 
             let yearly_inflation_rate = <YearlyInflation<T>>::get();
+            if yearly_inflation_rate.is_zero() {
+                return T::WeightInfo::handle_inflation(0u32);
```

Comment:
> I also wonder if some magic number should be used as weight instead here.

Why? What is a magic number for you?

Reply:
0e7e8350af6581ef4552f4266c61abe8229afe08

Reply:
The PR you linked doesn't seem to fix the issue.

Reply:
The sent commit was related to your comment "We need a test to ensure that inflation is noop if the inflation rate is not set."

Reply:
IDK, something like 1_000_000. The calculations that take place before the line where the function returns early do take some time, so it would be prudent to not put zero as the required weight.

```rust
T::WeightInfo::handle_inflation(0u32)
```

This is not zero weight. This is a lazy way of saying: Overestimate the weight. So I think this is somewhat of a magic number like you suggest.

Reply:
Ah, my bad, misread that :thumbsup: